### DISCUSSION TITLE
suport generators

### DISCUSSION THREAD

**Discussion by jogibear9988** [sebastienros/jint#1725] [Q&A] [answered]
is there any reason why generator functions are not yet supported?

**Comment by lahma** [comment]
I think the reason is the same for anything that has not  been implemented in OSS project.

**Comment by jogibear9988** [comment]
:-) 
I only thought maybe there is a blocker, that is more complicated than the other features, or smth. special.
Means only lack of time and need of somebody?

**Reply by lahma** [reply, marked as answer]
Requires changes to engine execution logic and adding support for better suspension of state. Much more complicated than regular library features. There's WIP PR for this #824 .

**Comment by jogibear9988** [comment]
Thanks for the info, I did miss there was a pull req. for the feature
